.ruff.toml

```diff
@@ -26,6 +26,7 @@ select = [
 #  "SIM",         # flake8-simplify
   "UP",          # pyupgrade
   "YTT",         # flake8-2020
+  "NPY201",      # numpy2-deprecation
```

Comment:
Since we are already specifying `NPY` above, I think this rule is already enabled by default.

See https://docs.astral.sh/ruff/rules/#numpy-specific-rules-npy